CMakeLists.txt

```diff
@@ -32,10 +32,14 @@ if(GREENTEA_CLIENT_STDIO)
     )
 endif()
 
-# Example application
+# Example applications
 
 set(BUILD_EXAMPLES ON CACHE BOOL "Enable building the examples")
 
 if((CMAKE_PROJECT_NAME STREQUAL PROJECT_NAME) AND BUILD_EXAMPLES)
-    add_subdirectory(example)
+    if(GREENTEA_CLIENT_STDIO)
+        add_subdirectory(examples/stdio)
+    else()
+        add_subdirectory(examples/custom_io)
+    endif()
```

Comment:
CMake question: Is it possible to build two applications at once, with two different configurations (GREENTEA_CLIENT_STDIO ON/OFF) of a static libraries? I know it's doable if both applications use same configuration of the library...

Reply:
I'm not aware of a way to do that elegantly. Inelegantly, you could write a Makefile which calls `cmake` with each set of configuration options and builds the targets you want. Another approach is to specify two different `build` directories, one for each configuration set, and then e.g. `cmake --build build_stdio` and `cmake --build build_nostdio`.

Reply:
Fair enough. I guess building two versions of the same target in one go is not a supported/intended use case of CMake. I'll keep this unchanged.